src/code/code_800F9280.c

```diff
@@ -2,9 +2,93 @@
 #include "global.h"
 #include "ultra64/abi.h"
 
-extern u8 sSeqCmdRdPos;
-extern u8 sSeqCmdWrPos;
-extern u8 D_80133410[];
+typedef struct {
+    u8 unk_0;
+    u8 unk_1; // importance?
+} Struct_8016E320;
+
+// bss
+Struct_8016E320 D_8016E320[4][5];
+u8 D_8016E348[4];
+u32 sAudioSeqCmds[0x100];
+unk_D_8016E750 D_8016E750[4];
+u8 D_8016F0E0[0xA0];
+AudioContext gAudioContext;
+void (*D_801755D0)(void);
+s32 D_801755D8[3]; // bss padding
+
+// data
+u8 sSeqCmdWrPos = 0;
+u8 sSeqCmdRdPos = 0;
+u8 D_80133408 = 0;
+u8 D_8013340C = 1;
+u8 D_80133410[] = { 0, 1, 2, 3 };
+u8 gAudioSpecId = 0;
+u8 D_80133418 = 0;
+u8 D_8013341C = 0; // pad
```

Comment:
is this pad needed?

Reply:
With the current setup, yes it is. However, the ReverbSettings and AudioSpec seem very out of place with the content of the file, very related to the 4 bss variables I commented on above. I wonder if there was another separate audio file for these variables. Maybe this could be file-padding too